_docs/overview/design-getting-started.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Getting Started
+layout: docs
+category: Overview
+usage: >
+  Read this article to learn how to use the Helix design language, which will help you design faster and provide a more consistent user experience.
+status: stable
+resource: true
+last-modified: 2017-10-24
+---
+
+
+## Prerequisites
+
+<div class="hxRow large">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+To take full advantage of Helix resources, you'll need Sketch. [Download Sketch here](http://www.sketchapp.com). Learn more about using Sketch with [these Pablo video lectures](https://www.youtube.com/watch?v=cEplnCnZuDM&list=PLWlUJU11tp4fEXI8deWhBQAHDv9R23WHB).
+
+{% endcolumn %}
+
+</div>
+
+
+## Get in the loop
+
+<div class="hxRow large">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+Join the following channels to talk to the Helix team and get updates.
+
+- Join the [#helix Slack channel](https://rackspace.slack.com/messages/#helix)
```

Comment:
Let's just make these items and not actions, since we call them to action with the introductory sentence.
- The #helix Slack channel
- The Helix mailing list
- The Helix GitHub repository

Reply:
updated.